helper_test.go

```diff
@@ -164,7 +164,7 @@ func createNotificationConfiguration(t *testing.T, client *Client, w *Workspace,
 			Name:            String(randomString(t)),
 			Token:           String(randomString(t)),
 			URL:             String("http://example.com"),
-			Triggers:        []string{NotificationTriggerCreated},
```

Comment:
I think someone at some point meant to create a typed string for the constants they created but they forgot to create the typed string and left it as a string.